blocks.js

```diff
@@ -2593,6 +2593,22 @@ BlockMorph.prototype.userMenu = function () {
         },
         'open a new window\nwith a picture of this script'
     );
+    menu.addLine();
+    menu.addItem(
+        'download stack',
```

Comment:
Do we use the term "stack" anywhere? Why not a "script"? (or am I misunderstanding something?)